server/webapp/WEB-INF/rails/webpack/models/users/users.ts

```diff
@@ -64,16 +69,20 @@ export class User {
   enabled: Stream<boolean>;
   checkinAliases: Stream<string[]>;
   roles: Stream<UserRoleJSON[]>;
+  updateOperationStatus: Stream<UpdateOperationStatus | null> = stream();
```

Comment:
This seems like a view state to me. Perhaps store this in a view model, or directly on the individual component if it works better.